README.md

```diff
@@ -130,7 +129,6 @@ module "example" {
 | ssm\_region | The region of the SSM to access. | `string` | `"us-east-1"` | no |
 | ssm\_tlscrypt\_key | The SSM key that contains the tls-auth key. | `string` | `"/openvpn/server/tlscrypt.key"` | no |
 | subnet\_id | The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0) | `string` | n/a | yes |
-| tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
 | trusted\_cidr\_blocks\_vpn | A list of the CIDR blocks that are allowed to access the VPN port on OpenVPN servers (e.g. ["10.10.0.0/16", "10.11.0.0/16"]) | `list(string)` | n/a | yes |
```

Comment:
Variable description would need to be updated.

```suggestion
| trusted\_cidr\_blocks\_vpn | A list of the CIDR blocks that are allowed to access the VPN port on OpenVPN servers (e.g. ["10.10.0.0/16", "10.11.0.0/16"]). | `list(string)` | n/a | yes |
```

Reply:
See commit f0aecc6.